src/main/java/ch/securify/decompiler/evm/OpCodes.java

```diff
@@ -166,9 +166,10 @@ public static int isSwap(int opcode) {
 	public static final int CALLCODE = 0xf2;
 	public static final int RETURN = 0xf3;
 	public static final int DELEGATECALL = 0xf4;
+	public static final int STATICCALL = 0xf5;	
```

Comment:
As per this yellow paper https://ethereum.github.io/yellowpaper/paper.pdf, STATICCALL code is 0xfa. 
Is there any other place to check for the latest opcodes of the instructions?